### PR TITLE
Feature/3720 export preview

### PR DESCRIPTION
**Pull request by cw-cvp**

### CODE REVIEW

File: src/components/export/ExportTab/ExportTablesContainer.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
+import { Preloader } from "@us-epa-camd/easey-design-system";
+import React, { useEffect, useState } from "react";
+import DataTable from "react-data-table-component";
+import { ArrowDownwardSharp } from "@material-ui/icons";
+
+import { getQATestSummary } from "../../../utils/api/qaCertificationsAPI";
+import { Button } from "@trussworks/react-uswds";
```
Comment (subrataroyCVP) [inline src/components/export/ExportTab/ExportTablesContainer.js]:
```suggestion
```

Looks like this not getting used. ⬆

File: src/components/export/ExportTab/ExportTablesContainer.js [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+        setQATestSummaryData(res.map((e) => e.data).flat());
+      });
+    };
+    fetchQATestSummary();
+  }, [beginDateOption, endDateOption]);
+
+  const rows = qaTestSummaryData?.map((obj) => {
+    const {
+      id,
+      componentID: componentId,
```
Comment (subrataroyCVP) [inline src/components/export/ExportTab/ExportTablesContainer.js]: You should be using `componentID` , not `componentId`. 

`componentID` stands for 'component_indetifier' and 
`componentId` stands for component record id.

File: src/components/export/ExportTab/ExportTablesContainer.js [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+    id: 1,
+    title: "Beetlejuice",
+    year: "1988",
+  },
+  {
+    id: 2,
+    title: "Ghostbusters",
+    year: "1984",
+  },
+];
```
Comment (subrataroyCVP) [inline src/components/export/ExportTab/ExportTablesContainer.js]: Cannot relate this.⬇

```suggestion
```

**Review by subrataroyCVP — CHANGES_REQUESTED** [review]
Everything Looks Good. Just few changes needed.

Also Fix the conflicts and run `yarn format` to format the code and It should be good.

File: src/components/export/ExportTab/ExportTablesContainer.js [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+  },
+  {
+    name: "End Date",
+    selector: (row) => row.endDate,
+  },
+  {
+    name: "End Hour",
+    selector: (row) => row.endHour,
+  },
+];
```
Comment (subrataroyCVP) [inline src/components/export/ExportTab/ExportTablesContainer.js]: This is getting duplicated you can just export `columns` form `src/components/QAImportHistoricalDataPreview/QAImportHistoricalDataPreview.js`

```suggestion
```

**Review by Qb10 — CHANGES_REQUESTED** [review]
you need to pull in latest and repush to avoid conflict

File: src/components/export/ExportTab/ExportTablesContainer.js (hunk 4 above)
Comment (cw-cvp) [reply src/components/export/ExportTab/ExportTablesContainer.js]: moved into src/utils/constants/tableColumns

**Pull request merged**